generator/tests/init-sources/assembly-example/configs/root-compilation.tsconfig.json

```diff
@@ -164,4 +164,4 @@
       "path": "../examples/assembly/compile.tsconfig.json"
     }
   ]
-}
+}
```

Comment:
please, revert it to have the checks passed

![image](https://user-images.githubusercontent.com/1636395/95347838-08da4080-08c6-11eb-851a-f37d4b54e79a.png)

Reply:
Done, thank you!